install/operator/pkg/syndesis/action/install.go

```diff
@@ -101,6 +101,22 @@ func (a *installAction) Execute(ctx context.Context, originalSyndesis *v1alpha1.
 	}
 	config[string(configuration.EnvOpenShiftOauthClientSecret)] = token
 
+	// Handle an external database being defined
+	externalDbSecName := syndesis.Spec.Components.Db.ExternalDBSecret
```

Comment:
I think you should just use an ExternalDbService or ExternalDbURL field to know if the mode is enabled.

Reply:
Do you mean include the entire connection string in the CR. I've not seen passwords be passed in via a CR before. The usual pattern I'd seen is that the CR references out to a secret if a password is needed.

If you're happy with the connection string being in the CR, I'd be happy to make that change

Reply:
I mainly meant the hostname:port mainly.  I think we have fields in the CR and our secrets to configure everything else needed for an external connection.

Reply:
Ok cool. I'll make changes to re-use as much of the existing CR/Secrets as possible instead of adding an extra Secret. Thanks for the review.